Jenkinsfile.integration

```diff
@@ -176,6 +177,19 @@ pipeline {
     }
 
     post {
+        always {
+            script {
+                sh 'ara generate html ara_report'
```

Comment:
I guess we need to make sure this command runs inside the appropriate venv?

Reply:
yes, this is something that I wanted to test with the jenkins run, as it may need tweaking (and also it may need ara install? Ara is probably not installed in the nodes and we are not using venv currently IIRC)